cunumeric/module.py

```diff
@@ -2079,12 +2079,14 @@ def array_split(
     """
     array = convert_to_cunumeric_ndarray(a)
     split_pts = []
-    if axis >= array.ndim:
-        raise ValueError(
-            f"array({array.shape}) has less dimensions than axis({axis})"
-        )
+    # if axis >= array.ndim:
+    #     raise ValueError(
+    #         f"array({array.shape}) has less dimensions than axis({axis})"
+    #     )
```

Comment:
What is the reason to leave in commented-out code?

Reply:
These codes caused different code messages that cuNumeric and numpy raised when axis is bigger than array.ndim. In my opinion, cunumeric handlers better because it makes a judgment on this part. So I opened these codes in my next commit.  What is your recommendation?

Reply:
FWIW I'm generally always -1 for leaving commented code in. If the code is really ever needed again in the future, that is what revision control is for. If it is left in there should be an accompanying comment explaining why it is there / what it is for (looks like you already removed it though)